sdk/search/azure-search/src/main/java/com/azure/search/models/ItemError.java

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents an item- or document-level indexing error.
+ */
+@Fluent
+public final class ItemError {
```

Comment:
ItemError vs. IndexerErorr?

Reply:
@careyjmac It's up to your team to make the call on that name. Just keep in mind that customers using the Track 1 .NET SDK already use the names `ItemError` and `ItemWarning`, so changing the name will cause some customer pain when adopting the new SDK.

Reply:
Not blocking for this PR.